dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -1,18 +1,28 @@
 ﻿#include "pch.h"
+
 #include "PushNotificationManager.h"
 #include "Microsoft.Windows.PushNotifications.PushNotificationManager.g.cpp"
+
 #include "PushNotificationCreateChannelResult.h"
+
+#include <winrt/Windows.ApplicationModel.background.h>
 #include <winrt\windows.networking.pushnotifications.h>
+#include "PushNotificationBackgroundTask.h"
+
 #include <winerror.h>
 #include <algorithm>
-#include <iostream>
+#include <vector>
```

Comment:
Where are we using this library here?

Reply:
Yep. I don't think this was ever being used.